src/simplify_algebra.cpp

```diff
@@ -268,6 +269,32 @@ struct find_dot_add
     }
 };
 
+struct find_conv_add
+{
+    auto matcher() const
+    {
+        auto add = match::name("add")(
+            match::either_arg(0, 1)(match::any().bind("x"),
+                                    match::any_of(match::is_constant()).bind("a")),
```

Comment:
constant `a` needs to be of same shape as `x`  or shape should be such that atleast `conv(x, w)` shouldn't fail.

Reply:
Nevermind, pointwise add would have both `a` and `x` guaranteed to be of the same type and dims.